src/string.cr

```diff
@@ -1694,6 +1750,20 @@ class String
     ary
   end
 
+  # Make an array by splitting the string on `separator` (and removing instances of `separator`).
+  # If `limit` is present, the array will be limited to `limit` items.
+  # The final item will contain the remainder of the string.
+  #
+  # If separator is a single space (`' '`, `" "`), it splits on any whitespace (see `#split()`).
+  #
+  # If the separator is `nil`, the string will be separated into one-character strings.
```

Comment:
What do you mean by that?

Oh, I see an error here. To get single-character strings, you should pass an empty string:

```
~/projects/crystal $ crystal eval
puts
puts "Mississippi".split("")
^D
["M", "i", "s", "s", "i", "s", "s", "i", "p", "p", "i"]
```